api/v1alpha1/flowcollector_types.go

```diff
@@ -35,6 +35,9 @@ type FlowCollectorSpec struct {
 
 	// Loki contains settings related to the loki client
 	Loki FlowCollectorLoki `json:"loki,omitempty"`
+
+	// ConsolePlugin contains settings related to the console dynamic plugin
+	ConsolePlugin FlowCollectorConsolePlugin `json:"consoleplugin,omitempty"`
```

Comment:
nit: could you rename the json field in camel case `consolePlugin` (and I just realize `goflowkube` is also not camel case maybe that's why you did that? i guess we'll have to rename it too, to stay consistent with the other fields)